examples/CMakeLists.txt

```diff
@@ -354,6 +373,41 @@ add_impactx_test(iotalattice.py.MPI
     OFF  # no plot script yet
 )
 
+# Kurth Beam in a Periodic Channel Test ###################################################
+#
+# w/o space charge
+add_impactx_test(kurth_periodic
+    examples/kurth/input_kurth_periodic.in
+      OFF  # ImpactX MPI-parallel
+      OFF  # ImpactX Python interface
+    examples/kurth/analysis_kurth_periodic.py
+    OFF  # no plot script yet
+)
+add_impactx_test(kurth_periodic.py
+    examples/kurth/run_kurth_periodic.py
+      OFF  # ImpactX MPI-parallel
+      ON   # ImpactX Python interface
+    examples/kurth/analysis_kurth_periodic.py
+    OFF  # no plot script yet
+)
+
+# w/ space charge
+add_impactx_test(kurth_10nC_periodic
+    examples/kurth/input_kurth_10nC_periodic.in
+      OFF  # ImpactX MPI-parallel
+      OFF  # ImpactX Python interface
+    examples/kurth/analysis_kurth_10nC_periodic.py
+    OFF  # no plot script yet
+)
+
+# w/ space charge
+add_impactx_test(kurth_10nC
+    examples/kurth/input_kurth_10nC.in
```

Comment:
Suggest that the test here, corresponding to "input_kurth_10nC.in", can be removed since the previous test "input_kurth_10nC_periodic.in" is more comprehensive.

Reply:
Thanks - removed :)